doc/source/getting_started/basics.rst

```diff
@@ -1937,21 +1937,39 @@ See :ref:`extending.extension-types` for how to write your own extension that
-period (time spans) :class:`PeriodDtype`      :class:`Period`    :class:`arrays.PeriodArray`   :ref:`timeseries.periods`
-sparse              :class:`SparseDtype`      (none)             :class:`arrays.SparseArray`   :ref:`sparse`
-intervals           :class:`IntervalDtype`    :class:`Interval`  :class:`arrays.IntervalArray` :ref:`advanced.intervalindex`
-nullable integer    :class:`Int64Dtype`, ...  (none)             :class:`arrays.IntegerArray`  :ref:`integer_na`
-Strings             :class:`StringDtype`      :class:`str`       :class:`arrays.StringArray`   :ref:`text`
-Boolean (with NA)   :class:`BooleanDtype`     :class:`bool`      :class:`arrays.BooleanArray`  :ref:`api.arrays.bool`
-=================== ========================= ================== ============================= =============================
++------------------+---------------------------+--------------------+-------------------------------+-----------------------------------------+-------------------------------+
+|Kind of Data      | Data Type                 | Scalar             | Array                         | String Shortcuts                        | Documentation                 |
++==================+===========================+====================+===============================+=========================================+===============================+
+|tz-aware datetime | :class:`DatetimeTZDtype`  | :class:`Timestamp` | :class:`arrays.DatetimeArray` | ``'datetime64[ns, <tz>]'``              | :ref:`timeseries.timezone`    |
++------------------+---------------------------+--------------------+-------------------------------+-----------------------------------------+-------------------------------+
+|Categorical       | :class:`CategoricalDtype` | (none)             | :class:`Categorical`          | ``'category'``                          | :ref:`categorical`            |
++------------------+---------------------------+--------------------+-------------------------------+-----------------------------------------+-------------------------------+
+|period            | :class:`PeriodDtype`      | :class:`Period`    | :class:`arrays.PeriodArray`   | ``'period[<freq>]'``,                   | :ref:`timeseries.periods`     |
+|(time spans)      |                           |                    |                               | ``'Period[<freq>]'``                    |                               |
++------------------+---------------------------+--------------------+-------------------------------+-----------------------------------------+-------------------------------+
+|sparse            | :class:`SparseDtype`      | (none)             | :class:`arrays.SparseArray`   | ``'Sparse'``, ``'Sparse[int]'``,        | :ref:`sparse`                 |
+|                  |                           |                    |                               | ``'Sparse[int32, 0]'``,                 |                               |
```

Comment:
I'm not sure we need all these. I think something like `Sparse[<numpy_dtype>]` like you do for datetime64 suffices. Perhaps include a concrete one like `Sparse[int64]`.

Reply:
I will put `Sparse[int]` and `Sparse[float]` in there, since  `Sparse[int, 0]` and `Sparse[float64, nan]` work, but only with the default fill values, so listing those fill values doesn't seem to add any value.